scripts/synology-lun-cleanup.py

```
@@ -0,0 +1,311 @@
+                return 0
+
+            if not args.yes:
+                if not Confirm.ask(
+                    f"\n[bold]Delete {len(matches)} LUN(s)?[/bold] (irreversible)",
+                    default=False,
+                ):
+                    console.print("aborted.")
+                    return 1
+
+            failures = []
+            for pv_name, lun_name, lun in matches:
```

Ruff (0.15.12)

[warning] Loop control variable `pv_name` not used within loop body

(B007)